### PR TITLE
Style/basic layout

### PR DESCRIPTION
**Pull request by trChris**
Added footer and responsive header with sidenav on mobile phones.
Screenshots: 
desktop
mobile
mobile-sidenav
de
mob

### CODE REVIEW

File: src/app/material.module.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import {
+  MatButtonModule, MatToolbarModule,
+  MatIconModule, MatListModule,
+  MatSidenavModule
+} from '@angular/material';
+import {LayoutModule} from '@angular/cdk/layout';
```
Comment (dmitryy90) [inline src/app/material.module.ts]: follow the same code style as everywhere

File: src/styles.scss [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+  height: 100%;
+  background-color: $light-color;
+}
 body {
-  background-color: #00caca;
-  font-family: Arial, Helvetica, sans-serif
+  margin: 0;
+  height: 100%;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
```
Comment (dmitryy90) [inline src/styles.scss]: please use relative dimension

Comment (trChris) [reply src/styles.scss]: Fixed and new review required

File: src/app/material.module.ts (hunk 1 above)
Comment (trChris) [reply src/app/material.module.ts]: Fixed

**Pull request merged**